libs/core/src/lib/notification/notification/notification.component.scss

```diff
@@ -1,621 +1,10 @@
-}
-.fd-notification--warning {
-    border-left-color: #e9730c;
-    border-left-color: var(--sapWarningBorderColor, #e9730c);
-}
-.fd-notification--warning[dir='rtl'],
-[dir='rtl'] .fd-notification--warning {
-    border-right-color: #e9730c;
-    border-right-color: var(--sapWarningBorderColor, #e9730c);
-}
-.fd-notification--information {
-    border-left-color: #0a6ed1;
-    border-left-color: var(--sapInformationBorderColor, #0a6ed1);
-}
-.fd-notification--information[dir='rtl'],
-[dir='rtl'] .fd-notification--information {
-    border-right-color: #0a6ed1;
-    border-right-color: var(--sapInformationBorderColor, #0a6ed1);
-}
-.fd-notification--m .fd-notification__body,
-.fd-notification--s .fd-notification__body {
-    -webkit-box-orient: vertical;
-    -webkit-box-direction: normal;
-    -ms-flex-direction: column;
-    flex-direction: column;
-    -webkit-box-align: start;
-    -ms-flex-align: start;
-    align-items: flex-start;
-    width: 100%;
-}
-.fd-notification--m .fd-notification__content,
-.fd-notification--s .fd-notification__content {
-    width: 100%;
-    padding: 1rem 1rem 0.5rem 1rem;
-}
-.fd-notification--m .fd-notification__content:last-child,
-.fd-notification--s .fd-notification__content:last-child {
-    padding-bottom: 1rem;
-}
-.fd-notification--m .fd-notification__footer,
-.fd-notification--s .fd-notification__footer {
-    width: 100%;
-}
-.fd-notification--m {
-    max-width: 640px;
-    min-width: 320px;
-}
-.fd-notification--s {
-    width: 320px;
-}
-.fd-notification--group > * {
-    border-radius: 0;
-}
-.fd-notification--group > :not(:first-child) .fd-notification__header {
-    -webkit-box-shadow: none;
-    box-shadow: none;
-}
-.fd-notification--group .fd-notification {
-    margin-bottom: 0;
-}
-.fd-notification--group .fd-notification:first-child {
-    border-radius: 0.25rem 0.25rem 0 0;
-}
-.fd-notification--group .fd-notification:last-child {
-    border-radius: 0 0 0.25rem 0.25rem;
-}
-.fd-notification--group .fd-notification:only-child {
-    border-radius: 0.25rem;
-}
-.fd-notification--group .fd-notification__content {
-    padding-top: 0.75rem;
-    padding-bottom: 0.5rem;
-}
-.fd-notification--group .fd-notification__title {
-    padding: 0.75rem 0;
-}
-.fd-notification--group .fd-notification__close {
-    top: 0.1875rem;
-    right: 0.1875rem;
-}
-.fd-notification--group[dir='rtl'] .fd-notification__close,
-[dir='rtl'] .fd-notification--group .fd-notification__close {
-    right: -0.1875rem;
-}
-.fd-notification--group .fd-notification__indicator--error,
-.fd-notification--group .fd-notification__indicator--information,
-.fd-notification--group .fd-notification__indicator--success,
-.fd-notification--group .fd-notification__indicator--warning {
-    padding-top: 0.75rem;
-    padding-bottom: 0.75rem;
+// To be removed with the new fund-styles release
+.fd-notification--group .fd-notification__body:last-child {
+    border-bottom: var(--sapList_BorderWidth) solid var(--sapList_BorderColor);
```

Comment:
This is fixed in PR in styles: https://github.com/SAP/fundamental-styles/pull/2272